onnxruntime/core/providers/acl/nn/conv.cc

```diff
@@ -0,0 +1,257 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Copyright (c) 2019, NXP Semiconductor, Inc. All rights reserved.
+// Licensed under the MIT License.
+
+#ifdef _WIN32
+#pragma warning(disable : 4244)
+#endif
+#include <thread>
+#include <mutex>
+
+#include "core/common/common.h"
+#include "core/framework/op_kernel.h"
+#include "core/util/math.h"
+#include "core/util/math_cpuonly.h"
+
+#include "core/providers/acl/nn/conv.h"
+#include "core/providers/acl/acl_common.h"
+#include "core/providers/acl/acl_fwd.h"
+
+// ACL
+#include "arm_compute/core/TensorInfo.h"
+
+// NEON
+#include "arm_compute/runtime/NEON/functions/NEConvolutionLayer.h"
+#include "arm_compute/runtime/NEON/functions/NEDepthwiseConvolutionLayer.h"
+
+#define CONV_ACL
+#define DEPTHWISE_CPU
+
+#define PREF_DIM 4
```

Comment:
should have a way to set these at compile time instead of change code in the file. also put them in cmakelist?